### ISSUE TITLE
imgui.ini doesn't track which docknode was last open

### ISSUE THREAD

**Issue by Folling** [ocornut/imgui#3049]
**Version/Branch of Dear ImGui:**

Version: 1.74 WIP
Branch: docking

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui_impl_glfw.cpp + imgui_impl_opengl3.cpp
Operating System: Arch Linux 64bit

**My Issue/Question:**
I store my docking layout using the `imgui.ini` which I found wonderfully easy. However having multiple nodes opened as tabs does not properly store which node was last open in the imgui.ini, which I would find more appropriate behaviour.

**Screenshots/Video**
[short video demonstrating the issue and desired solutions](https://www.youtube.com/watch?v=Oc6W6mkhu7Y)

I hope this demonstrates the issue nicely. It's hard to provide example code, I'm really just using a docking setup, and having it be loaded via the ini, which looks like this:

<details>
<summary>imgui.ini</summary>
[Window][DockSpaceViewport_11111111]
Pos=0,0
Size=1280,720
Collapsed=0

[Window][Statistics]
Pos=841,0
Size=439,720
Collapsed=0
DockId=0x00000002,0

[Window][Debug##Default]
Pos=347,186
Size=400,400
Collapsed=0

[Window][Production]
Pos=0,0
Size=839,720
Collapsed=0
DockId=0x00000001,0

[Window][Upgrades]
Pos=0,0
Size=839,720
Collapsed=0
DockId=0x00000001,1

[Window][Research]
Pos=0,0
Size=839,720
Collapsed=0
DockId=0x00000001,3

[Window][Achievements]
Pos=0,0
Size=839,720
Collapsed=0
DockId=0x00000001,4

[Window][Skills]
Pos=0,0
Size=839,720
Collapsed=0
DockId=0x00000001,2

[Window][Settings]
Pos=841,0
Size=439,720
Collapsed=0
DockId=0x00000002,1

[Window][Test]
Pos=60,60
Size=295,298
Collapsed=0

[Window][window]
Pos=511,144
Size=508,399
Collapsed=0

[Window][Main]
Size=1048,451
Collapsed=0

[Window][Overarching Main]
Size=1280,720
Collapsed=0

[Docking][Data]
DockSpace   ID=0x8B93E3BD Window=0xA787BDB4 Pos=0,0 Size=1280,720 Split=X
  DockNode  ID=0x00000001 Parent=0x8B93E3BD SizeRef=839,720 CentralNode=1 Selected=0xD29D1346
  DockNode  ID=0x00000002 Parent=0x8B93E3BD SizeRef=439,720 Selected=0xAD8E88F2
</details>

**Comment by ocornut** [comment]
Hello,

This is duplicate of #2507 and #2304
TL;DR; You are right this is an issue but it's not particularly specific to Docking. We don't track the focus and Z-order.

Closing to move back to #2304
